packages/core/src/index.ts

```diff
@@ -1,4 +1,4 @@
-export type { ClientClass } from './sdk';
+export type { ClientClass as SentryCoreCurrentScopes } from './sdk';
```

Comment:
m: is this on purpose? 😅 seems unintentional (and probably breaking, technically...)